packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -227,6 +236,46 @@ class UiKitView extends StatefulWidget {
   /// This must not be null if [creationParams] is not null.
   final MessageCodec<dynamic> creationParamsCodec;
 
+  /// Which gestures should be forwarded to the UIKit view.
+  ///
+  /// {@macro flutter.widgets.platformViews.gestureRecognizersDescHead}
+  ///
+  /// For example, with the following setup vertical drags will not be dispatched to the UIKit
+  /// view as the vertical drag gesture is claimed by the parent [GestureDetector].
+  /// ```dart
+  /// GestureDetector(
+  ///   onVerticalDragStart: (DragStartDetails d) {},
+  ///   child: UiKitView(
+  ///     viewType: 'webview',
+  ///   ),
+  /// )
+  /// ```
+  /// To get the [UiKitView] to claim the vertical drag gestures we can pass a vertical drag
+  /// gesture recognizer factory in [gestureRecognizers] e.g:
+  /// ```dart
+  /// GestureDetector(
+  ///   onVerticalDragStart: (DragStartDetails d) {},
+  ///   child: SizedBox(
+  ///     width: 200.0,
+  ///     height: 100.0,
+  ///     child: UiKitView(
+  ///       viewType: 'webview',
+  ///       gestureRecognizers: <Factory<OneSequenceGestureRecognizer>>[
```

Comment:
I feel like specifying a `Map<Type, GestureRecognizerFactory>` is pretty inconvenient as the highest level API surface. RawGestureDetector isn't likely to be used frequently by app developers, where this thing will be specified directly by app developers.
I plan to eventually add some pre made factories to the framework so using this could look something like:

```
gestureRecognizers: <Factory<OneSequenceGestureRecognizer>> [
  VerticalDragRecognizerFactory(),
  ScaleRecognizerFactory(),
],
```